src/pages/commits/ActiveCommits/ChartToggle.tsx

```diff
@@ -17,40 +17,43 @@ export const ChartToggle: React.VFC<{
     onChangeChartType(chartType);
   };
   return (
-    <ClassNames>
-      {({ css }) => (
-        <ToggleWrapper>
-          <Label htmlFor="chart-toggle">View Options</Label>
-          <StyledRadioGroup
-            size="default"
-            onChange={onChange}
-            value={currentChartType}
-            name="chart-select"
-            className={css`
-              font-weight: bold;
-            `}
-          >
-            <Radio
-              data-cy="cy-chart-absolute-radio"
-              id="chart-radio-absolute"
-              value={ChartTypes.Absolute}
-            >
-              Absolute Number
-            </Radio>
-            <Radio
-              data-cy="cy-chart-percent-radio"
-              id="chart-radio-percent"
-              value={ChartTypes.Percentage}
-              className={css`
-                font-weight: bold;
-              `}
-            >
-              Percentage
-            </Radio>
-          </StyledRadioGroup>
-        </ToggleWrapper>
-      )}
-    </ClassNames>
+    <AccordionContainer>
+      <Accordion title="View options">
```

Comment:
I think this should be expanded by default. So it's visible to the user.

Reply:
I kind of appreciate that it's closed by default, since the user probably won't change this very often

If it was always open it could be annoying when it blocks the graph bars (the component gets completely re-rendered after fetching data, so the open/close state isn't preserved)

https://user-images.githubusercontent.com/47064971/162991718-ff596504-d38f-46ea-b347-72b853b826c7.mov

Reply:
Could we save its state as a cookie? I think it makes sense to at least show it by default so new users are aware of it and they can dismiss it on their own. We can discuss it in the design sync as well.